ios/Client/EXAppDelegate.m

```diff
@@ -41,9 +45,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 #pragma mark - Handling URLs
 
-- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url sourceApplication:(nullable NSString *)sourceApplication annotation:(id)annotation
+- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options
 {
-  return [[ExpoKit sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation];
+  id annotation = options[UIApplicationOpenURLOptionsAnnotationKey];
+  NSString *sourceApplication = options[UIApplicationOpenURLOptionsSourceApplicationKey];
+#if __has_include(<GoogleSignIn/GoogleSignIn.h>)
```

Comment:
I think it is time to create some universal modules-compatible integration with ExpoKit and AppDelegate methods…

Reply:
@sjchmiela - could you possibly create an issue to track that? I agree this isn't ideal but also think this is an okay solution for now if we have a clear plan to improve it in the near future.